services/web/client/source/class/osparc/component/form/renderer/PropFormBase.js

```diff
@@ -281,7 +289,9 @@ qx.Class.define("osparc.component.form.renderer.PropFormBase", {
         });
         osparc.data.model.Sweeper.isSweeperEnabled()
           .then(isSweeperEnabled => {
-            menuBtn.setVisibility(isSweeperEnabled ? "visible" : "excluded");
+            field.bind("visibility", menuBtn, "visibility", {
+              converter: visibility => (visibility === "visible" && isSweeperEnabled) ? "visible" : "excluded"
```

Comment:
Q: Does "visibility" has other states than ``visible`` and ``not visible``?

Reply:
```visible```, ```hidden``` or ```excluded```.
```excluded``` ignores the widget during the layout process while ```hidden``` simply
hides the widget and keeps the room for the widget during the layout process.